examples/nats-payments/gen.py

```
@@ -0,0 +1,184 @@
+                amount = round(random.uniform(5.0, 2000.0), 2)
+                evt = {
+                    "payment_id": pid,
+                    "region":     region,
+                    "method":     method,
+                    "amount_usd": amount,
+                    "event_time": now_iso(),
+                }
+                await nc.publish(PAYMENT_SUBJECT, json.dumps(evt).encode())
+
+                # Schedule the fraud score follow-up.
+                roll = random.random()
```

Ruff (0.15.11)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)